wazo_confgend/generators/extensionsconf.py

```diff
@@ -183,7 +183,7 @@ def generate(self, output):
             context_name = ctx['name']
             if conf and conf.has_section('!%s' % context_name):
                 continue
-
+            ast_writer.write_newline()
```

Comment:
With this change we have now 4 newlines between each context section :sweat_smile: